src/components/Navigation/DropdownNav.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { within, userEvent } from "@storybook/test";
+import type { DropdownNavGroup } from "./DropdownNav";
+import { DropdownNav } from "./DropdownNav";
+
+const meta = {
```

Comment:
@winkerVSbecks, one item here, as you introduced Chromatic's modes feature, it now looks like we introduced a regression. Checking the latest build in Chromatic and navigating to this [component's story](https://66a7c58b7ecbd7bacf6a2264-znhbdnnbcj.chromatic.com/?path=/story/components-dropdownnav--collapsed) there's an error with the interaction test.